.ci/test_r_package.sh

```diff
@@ -241,8 +241,13 @@ if [[ $R_BUILD_TYPE == "cmake" ]]; then
     fi
 fi
 
-
-if grep -q -E "NOTE|WARNING|ERROR" "$LOG_FILE_NAME"; then
+# ignoring the following NOTE:
+#
+# * checking C++ specification ... NOTE
+#   Specified C++11: please update to current default of C++17
+#
+# until it's resolved (see https://github.com/microsoft/LightGBM/pull/5690)
```

Comment:
proposing adding this to get CI working again, to work around this new NOTE that started showing up with r-devel two days ago.

```text
* checking C++ specification ... NOTE
   Specified C++11: please update to current default of C++17
```

Put up #5690 with a proposal to handle that and a lot more information.